src/Datadog.Trace.ClrProfiler.Managed/Integrations/AWS/AmazonSQSIntegration.cs

```diff
@@ -0,0 +1,254 @@
+using System;
+using System.Threading.Tasks;
+using Datadog.Trace.ClrProfiler.Emit;
+using Datadog.Trace.Logging;
+
+namespace Datadog.Trace.ClrProfiler.Integrations
+{
+    /// <summary>
+    /// Tracing integration for AWSSDK.SQS.
+    /// </summary>
+    public static class AmazonSqsIntegration
+    {
+        private const string IntegrationName = "AWS";
+        private const string OperationName = "aws.command";
+        private const string AgentName = "dotnet-aws-sdk";
+
+        private const string Major3 = "3";
+        private const string Major3Minor3 = "3.3";
```

Comment:
I would rename these entries to `MinimumSupportedVersion_Major` and `MinimumSupportedVersion_Minor` or something like that.

Reply:
This is the format we're using for all the integrations. I'd prefer to keep it the same unless we decide to standardize on another format.

Reply:
That makes no sense to me. The symbolic name should represent what the intent is. Anyway, that's fine, maybe I'll change them all in a separate PR :)

Reply:
A little knot in that might be that we may support several version ranges within one integration.
The names might get excessively verbose.
It's fine with me though.

Reply:
To me, having `Major4 = "4"` just gives you another place to make a mistake. Copy and paste and you get `Major4 = "5"`.

Reply:
Bob makes some good points, I think this worth revisiting, though it doesn't necessarily need to be addressed in this PR